apps/onlyoffice.sh

```diff
@@ -23,6 +23,51 @@ root_check
 # Nextcloud 13 is required.
 lowest_compatible_nc 13
 
+## Onlyoffice for NC 18.0.1 onwwards
+# Check for the NC-version
+CURRENTNCVERSION="$(occ_command config:system:get version | sed 's/^\(.*\..*\..*\)\..*/\1/')"
+CURRENTNCVERSION="${CURRENTNCVERSION//./}"
+if [ "$CURRENTNCVERSION" -lt "9999" ]
+then
+    CURRENTNCVERSION="$(sed 's/./&0/3' <<<"$CURRENTNCVERSION")"
+fi
+
+# Check if onlyoffice is already installed
+print_text_in_color "$ICyan" "Checking if Onlyoffice or Collabora is already installed..."
+if [ "$CURRENTNCVERSION" -ge 18001 ] && ! does_this_docker_exist 'onlyoffice/documentserver' || ! does_this_docker_exist 'collabora/code'
```

Comment:
This then checks reliably, if the version is 18.0.1 or higher.

Reply:
Checking for only 18 or higher is what we need.

Reply:
no, that is not right. as I stated above.